public/Leo Jayachandran/Glossarizer/libretextsGlossarizer.js

```diff
@@ -12,8 +12,11 @@ class LibreTextsGlossarizer {
             /* Replace only once in a TextNode */
             replaceClass: 'glossarizer_replaced',
             caseSensitive: false,
-            exactMatch: false
+            exactMatch: true
         };
+        if (typeof(localStorage.glossarizerType) == "undefined" || localStorage.getItem("glossaryType") === null) {
```

Comment:
These keys do not match. If this is not intentional, the fix would be:
```suggestion
        if (typeof(localStorage.glossarizerType) == "undefined" || localStorage.getItem("glossarizerType") === null) {
```

Reply:
Also here is a stronger check for the localStorage item if necessary:
```suggestion
        if (typeof(localStorage.glossarizerType) == "undefined" || !localStorage.getItem("glossarizerType")) {
```

Reply:
These changes have been made in the latest commit